filters/encrypt/tag.go

```diff
@@ -0,0 +1,81 @@
+package encrypt
+
+import (
+	"reflect"
+	"strings"
+)
+
+type tagInfo struct {
+	Classification DataClassification
+	Operation      FilterOperation
+}
+
+func getClassificationFromTag(f reflect.StructTag, opt ...Option) *tagInfo {
+	t, ok := f.Lookup(DataClassificationTagName)
+
+	if !ok {
+		return &tagInfo{
+			Classification: UnknownClassification,
+			Operation:      UnknownOperation,
+		}
+	}
+	return getClassificationFromTagString(t, opt...)
+}
+
+func getClassificationFromTagString(tag string, opt ...Option) *tagInfo {
```

Comment:
There's a lot of evaluation going on here, I feel like a comment breaking it down would make it feel less intimidating

Reply:
added a godoc comment.